sources/safe/arbitrum/safe_arbitrum_sources.yml

```
@@ -29,4 +29,34 @@ sources:
             description: "Address of created Safe proxy"
           - &singleton
             name: singleton
-            description: "Singleton"
\ No newline at end of file
+            description: "Singleton"
+
+  - name: gnosis_safe_arbitrum
+    freshness:
+      warn_after: { count: 12, period: hour }
+    tables:
+      - name: ProxyFactoryv_1_4_1_evt_ProxyCreation
+        loaded_at_field: evt_block_time
+        description: "Safe v1.4.1 factory ProxyCreation events"
+        columns:
+          - &contract_address
+            name: contract_address
+            description: "Contract address"
```

Since `contract_address` has been defined in L12,
we can just use `*contract_address`.
```suggestion
          - *contract_address
```